netebpfext/net_ebpf_ext.h

```diff
@@ -161,7 +185,8 @@ net_ebpf_extension_get_callout_id_for_hook(net_ebpf_extension_hook_id_t hook_id)
  * @param[in]  condition_count Count of filter conditions.
  * @param[in]  conditions Common filter conditions to be applied to each filter.
  * @param[in]  raw_context Caller supplied context to be associated with the WFP filter.
- * @param[out] filter_ids Output buffer where the added filter IDs are stored.
+ * @param[out] filter_instances Output buffer to store the pointers to the filter instances
```

Comment:
```suggestion
 * @param[out] filter_ids Output buffer to store the pointers to the filter instances
```
match line 201 or doxygen will generate warnings

Reply:
Fixed